### DISCUSSION TITLE
Cascader级联选择在多选的时候启用了搜索，选中某搜索结果之后，就清空了搜索

### DISCUSSION THREAD

**Discussion by ekuku1984** [ant-design/ant-design#43951] [Ideas]
Cascader级联选择在多选的时候启用了搜索，选中某搜索结果之后，就清空了搜索，能不能做个选项，在多选的情况下，选中某个搜索结果的记录后不清空继续，因为使用搜索后，可能需要选择多个， 现在是选择一下就清空了搜索，返回到了正常的选择界面

**Comment by linxianxi** [comment]
https://github.com/ant-design/ant-design/pull/44033 5.9.0 可实现
